src/tokenize_utils.jl

```diff
@@ -4,12 +4,21 @@ const EOF_CHAR = typemax(Char)
 
 function is_identifier_char(c::Char)
     c == EOF_CHAR && return false
-    return Base.is_id_char(c)
+    return try
```

Comment:
Might be faster to use `Base.ismalformed()` here as a pre-check rather than the `try ... catch` ?